Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts
+FROM node:14.18.1
```

Comment:
Is there a reason why would we need to pin it to patch level granularity, in general?

I understand that after `14.18.1`, the version was bumped up to 15 an so on.

Reply:
So we are following the LTS release(Long term support), and the latest version of it is what @onuzbee has added which is 14.18.1, once the newer version starts supporting LTS, we would then have to update it, but for now the latests is for [14.18.1](https://nodejs.org/en/).

Reply:
++